src/Shared/XUnitLogger.cs

```diff
@@ -84,7 +97,14 @@ private XUnitLogger(string name, XUnitLoggerOptions? options)
     /// <summary>
     /// Gets or sets a delegate representing the system clock.
     /// </summary>
-    internal Func<DateTimeOffset> Clock { get; set; } = static () => DateTimeOffset.Now;
+    internal Func<DateTimeOffset> Clock { get; set; } = static () =>
```

Comment:
I'm fairly sure this won't compile in the .NET 8 case as the `static` will prevent `_timeProvider` from being accessible.